Legge til nye NRK-logoer

stianlj opened the pull request:
Se: https://github.com/nrkno/core-icons/issues/343

stianlj committed:
feat: update logo files and move old versions to deprecated folder

- Move outdated logo SVG files to lib/deprecated/ directory
- Add new on-dark, on-light, and with-bg variants for NRK logos
- Update existing logo SVG files with new designs

stianlj committed:
fix: remove width and height attributes from SVG files

gesi commented:
Har det blitt noe kluss med `viewBox`-en i disse SVG-ene eller er det GitHub sin preview som viser feil?
<img width="782" alt="Screenshot 2025-04-02 at 14 46 19" src="https://github.com/user-attachments/assets/9c87c863-7743-4ce7-bad7-b5b3c2d303cf" />

gesi commented:
> Har det blitt noe kluss med `viewBox`-en i disse SVG-ene 

Ser ut som at det blir plukka opp i bygget

<img width="751" alt="Screenshot 2025-04-02 at 14 48 48" src="https://github.com/user-attachments/assets/6df1b375-0854-44d7-9f4d-fde9d730301a" />

stianlj commented:
@gesi Jepp, jeg plukka også opp det etter at jeg laget pull requesten.

stianlj committed:
fix: add viewBox attribute to SVG logo files

gesi commented:
Jeg er litt usikker på om `viewBox`-ene har blitt riktig?

For eksempel: stemmer det at NRK 2-logoen med og uten bakgrunn skal ha forskjellige størrelser, og at de skal ha så mye (og ubalansert) luft rundt?

cc @asmundmatzow 

![Screenshot 2025-04-03 at 08 32 35](https://github.com/user-attachments/assets/37687388-6f14-4027-aa66-6c9faf5041a5)

![Screenshot 2025-04-03 at 08 32 43](https://github.com/user-attachments/assets/c18d0a96-320f-4a34-a6c9-84552bc7cd37)

stianlj committed:
fix(logo): correct SVG viewBox and position attributes for NRK logo

- Updated viewBox attribute from "0 0 55 24" to "0 0 54 24"
- Changed path fill attribute from ".087" to "0" for better precision

stianlj committed:
fix: adjust SVG viewports and path coordinates for NRK logos
